src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -211,6 +210,27 @@ def create(input_dataset, target, feature=None, validation_set='auto',
 
     train_accuracy = _mx.metric.Accuracy()
     validation_accuracy = _mx.metric.Accuracy()
+    
+    def compute_validation_accuracy(validation_accuracy):
+        validation_loader = _SFrameClassifierIter(validation_dataset, batch_size,
```

Comment:
You should also be able just to create the iterator once and call `reset` on it, I think

Reply:
Fixed.